src/InteractiveWindow/Editor/Commands/ClearScreenCommand.cs

```diff
@@ -20,12 +17,13 @@ public override Task<ExecutionResult> Execute(IInteractiveWindow window, string
 
         public override string Description
         {
-            get { return "Clears the contents of the REPL editor window, leaving history and execution context intact."; }
+            // TODO: Needs localization...
```

Comment:
Please open an issue for it else we might miss it if we don't intend to address all TODOs in 1.1 milestone